### DISCUSSION TITLE
Newbie help

### DISCUSSION THREAD

**Discussion by cheetos185** [quietvoid/dovi_tool#269] [Q&A]
Hello,

I want to convert a BL+EL+RPU MKV file to BL+RPU and i am using windows can you please explain to me how do i setup. I downloaded the latest zip file and read the readme file in it but it has no instructions on how to use it. I had an older version of this tool that came with ffmpeg mkvtools and it was very straight forward easy to use but this one is very different it has .rs file extension.

**Comment by quietvoid** [comment]
Sounds like you downloaded the source code and not the executables.

If you're on Windows it would probably be simpler to use helper scripts such as https://github.com/R3S3t9999/DoVi_Scripts.
I don't provide support for those.

**Reply by cheetos185** [reply]
thank you much appreciated
